test/integration/things-media.js

```diff
@@ -0,0 +1,201 @@
+const CONFIG = require('config')
+const __ = CONFIG.universalPath
+const _ = __.require('lib', 'utils')
+
+require('should')
+
+const breq = require('bluereq')
+const get = (url) => breq.get(url).then(_.property('body'))
+const post = (url, body) => breq.post(url, body).then(_.property('body'))
+const put = (url, body) => breq.put(url, body).then(_.property('body'))
+const delete_ = (url) => breq.delete(url).then(_.property('body'))
+
+const endpoint = CONFIG.server.url() + '/media'
```

Comment:
I do not see the **uuid** of the POI the media file(s) are assigned to anywhere. 

- Should not there be a reference to it so the uploaded/retrieved media(s) can be related?

As in, if I POST a new media file stored on ipfs ( for example) by POSTing a json file similar to test/fixtures/media-new-ipfs.json to https://data.transformap.co/media/, how does the API know which POI I want to upload it to?